libs/facilities/frontend/view/src/lib/components/browse/browse.page.ts

```diff
@@ -1,13 +1,36 @@
 import { CommonModule } from '@angular/common';
-import { ChangeDetectionStrategy, Component, ViewEncapsulation } from '@angular/core';
+import {
+	ChangeDetectionStrategy,
+	Component,
+	Input,
+	OnInit,
+	ViewEncapsulation,
+} from '@angular/core';
+import { RouterLink } from '@angular/router';
+import { IxModule } from '@siemens/ix-angular';
+
+import { facilities, Facility } from '../facility';
 
 @Component({
 	selector: 'lib-browse',
```

Comment:
Pls change selector to `xd-browse-page`

Reply:
It has to start with lib, I could also write //eslint-disable but I am not sure if that is better
![grafik](https://github.com/amosproj/amos2024ss01-xcelerator-demo-app/assets/166606862/8a7b1ca8-89c8-4025-ae8a-0784bf5c5e87)